app/views/services/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :page_title, @service.label %>
+
+<% if flash[:success] %>
+<div class="alert alert-success"><%= flash[:success] %></div>
+<% end %>
+
+<% breadcrumb :service, @service %>
+
+<div class="page-title">
+  <h2><%= @service.label %></h2>
+  <p><b>Service code</b> <%= @service.lgsl_code %></p>
+</div>
+
+<% if @links.any? %>
+  <table class="table table-bordered table-service">
+    <thead>
+      <tr class="table-header">
+        <th class='center'>Code</th>
+        <th width:'40%'>Council and links</th>
+        <th>Link status</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @local_authorities.each do |local_authority| %>
+        <% links_by_authority = @links[local_authority.id] %>
+        <% next if links_by_authority.blank? %>
+        <% presented_authority = LocalAuthorityPresenter.new(local_authority) %>
+        <tr data-local-authority-id="<%= presented_authority.id %>" class="header-row">
+          <th class="name" colspan="4">
+            <h3><%= link_to presented_authority.name, local_authority_path(presented_authority.slug) %></h3>
+          </th>
+        </tr>
+        <% first_link, *remaining_links = *links_by_authority %>
```

Comment:
This could just be `first_link, *remaining_links = links_by_authority` as the splat on `links_by_authority` is redundant.  The splat on `remaining_links` means to collect all the rest of the values.  E.g.:

```ruby
first, rest = [1,2,3,4,5]
first #=> 1
rest #=> 2

first, rest = [1,2,3,4,5]
first #=> 1
rest #=> [2,3,4,5]
```